### ISSUE TITLE
Update the app in Google Play

### ISSUE THREAD

**Issue by emanchado** [NewProggie/SparkleShare-Android#82]
According to Google Play, the app was last updated over one year ago. I'm not sure how many fixes there are now compared to the version in Google Play, but at least #52 seems pretty important to release.

Are there any plans to update the app in the market soon?

**Comment by NewProggie** [comment]
Yes, thanks for the reminder. I will push an update in the upcoming days.

**Comment by mbinz** [comment]
Hi,
I just wanted to know if this was already updated in play store? I would really love to see this working.
Thanks for the great work!

**Comment by mrtorrent** [comment]
@NewProggie Ping. Can't find this in the Play Store now. Has it been removed?

**Comment by Sixthhokage1** [comment]
https://github.com/NewProggie/SparkleShare-Android/issues/83#issuecomment-15164491

**Comment by MartinDelille** [comment]
The android market link is still dead. Is the project still alive?

**Comment by NewProggie** [comment]
Hey,
guess so. You can still build the project yourself though.
